react-native/react/engine/transport.desktop.js

```diff
@@ -14,17 +25,12 @@ export default class DesktopTransport extends BaseTransport {
       console.error('No keybased socket file found!')
     }
 
-    let hooks = null
-    if (connectCallback) {
-      hooks = {connected: connectCallback}
-    }
+    this.net_opts.path = sockfile
 
-    super(
-      {path: sockfile, hooks},
-      null,
-      incomingRPCCallback
-    )
-    this.needsConnect = true
-    this.needsBase64 = false
+    if (sockfile) {
+      super._connect_critical_section(cb)
+    } else {
+      cb(new Error('No socketfile'))
```

Comment:
so whoever calls this knows to retry when it gets an error in the callback?

where does the retry happen?